examples/PlutusComparison.md

```diff
@@ -0,0 +1,288 @@
+
```

Comment:
Please move this file to a `doc` directory in the repo root. Also, I would change the name of file to something like `side-by-side-comparison.md`, since you're directly comparing two programs